static/lang/re-en.json

```diff
@@ -2403,6 +2403,15 @@
                 },
                 "Overdrive": {
                     "DamageTypePrompt": "Deal fire damage with overdrive?"
+                },
+                "Unstable": {
+                    "FlatCheck": {
+                        "CriticalFailure": "The innovation becomes incapable of being used for further unstable actions, and you also take [[/r @actor.level[fire]]] damage.",
```

Comment:
Should this use `@Damage`?

Reply:
It can, but it shouldn't be modifiable in any way. This is the simplest way of pulling that off.